source/views/sis/course-search/search.js

```diff
@@ -114,6 +120,28 @@ function executeSearch(args: {
 	}
 }
 
+function executeBrowse(args: {
```

Comment:
Does a name like "applyFiltersAndQuery" fit this function better than "executeBrowse"?